.env.example

```diff
@@ -3,3 +3,7 @@ L2_WEB3_ENDPOINTS=
 
 # wallet private key
 PRIVATE_KEY=
+
+STAGE=production
+NETWORK=optimism
+AWS_REGION=
```

Comment:
can add a comment here to emphasize this is optional and only for deployment.

Reply:
fixed